src/main/java/ticTacToe/game/Mark.java

```diff
@@ -2,18 +2,36 @@
 
 public enum Mark {
 
-    NOUGHT("O"), CROSS("X");
+    NOUGHT("O"),
+    CROSS("X");
 
-    Mark(String name) {
-        mark = name;
+    Mark(String mark) {
+        sign = mark;
    }
 
-    public String mark;
+    public final String sign;
 
-    public static String switchPlayerMark(String currentMark) {
-        if (currentMark.equals(CROSS.mark)) {
-            return NOUGHT.mark;
+    public static boolean isValidMark(String markChosen) {
```

Comment:
Since this and `createMark` are static, the most straightforward way to call them is
```java
if(Mark.isValidMark(thing)) {...}
```
It feels to me like we are repeating the word `Mark` too often.
Would `Mark.isValid(thing)` convey the same meaning?

Reply:
Yes!